scripts/tcpdirect_misc/tcpdirect.spec

```diff
@@ -1,6 +1,13 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: (c) 2016-2023 Advanced Micro Devices, Inc.
 ######################################################################
+# RPM spec file for TCPDirect
+#
+# To build a source RPM:
+#   scripts/zf_make_official_srpm --version <version>
+#
+# To build a binary RPM from a source RPM:
+#   rpmbuild --define "onload_tarball <onload_tarball>" --rebuild ~/rpmbuild/SRPMS/tcpdirect-<version>-1.src.rpm
 
 %define _unpackaged_files_terminate_build 0
 pkgversion-DEFINITION
```

Comment:
Thanks, it sounds good. I will apply your suggestion in the next update of the PR.

(I looked at it and didn't like it for the same reason, but didn't fix it because it wasn't on my way!)